### ISSUE TITLE
preset: test failure with no test output.

### ISSUE THREAD

**Issue by evoskuil** [libbitcoin/libbitcoin-network#407]
```
Running tests...
/usr/local/bin/ctest --force-new-ctest-process 
Test project /home/runner/work/libbitcoin-network/libbitcoin-network/obj/nix-gnu-debug-shared
    Start 1: libbitcoin-network-test
1/1 Test #1: libbitcoin-network-test ..........***Failed    0.26 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.33 sec

The following tests FAILED:
	  1 - libbitcoin-network-test (Failed)
Errors while running CTest
Output from these tests are in: /home/runner/work/libbitcoin-network/libbitcoin-network/obj/nix-gnu-debug-shared/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
make: *** [Makefile:71: test] Error 8
```
```
Run cat /home/runner/work/libbitcoin-network/libbitcoin-network/Testing/Temporary/LastTest.log
cat: /home/runner/work/libbitcoin-network/libbitcoin-network/Testing/Temporary/LastTest.log: No such file or directory
```
https://github.com/libbitcoin/libbitcoin-network/actions/runs/9195550780/job/25291624293?pr=406

**Comment by evoskuil** [comment]
The test failure went away upon restarting the job, which is potentially problematic - unless both issues are actually a consequence of an underlying build platform issue. It's worth posting an intentional test failure to find out.

**Comment by pmienk** [comment]
Resolved by libbitcoin/libbitcoin-build#324, specifically libbitcoin/libbitcoin-build@232b450ab787264b6c85c625d3d692768173ddfa
